app/users/__init__.py

```diff
@@ -41,3 +42,19 @@ def post(self):
             return utils.make_validation_error(e)
         
         return utils.make_response(data=user.to_dict())
+
+    @authenticated_request
+    def get(self, id = None, user = None):
+        """
+        Returns the requested user
+        """
+
+        user = user_service.find_by_id(id)
```

Comment:
I just noticed that we declared `user` in the method definition. Let's rename this to `requested_user` so that people looking at this don't get confused.